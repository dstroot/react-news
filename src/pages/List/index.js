import React, { useEffect } from "react";

// components
import EditList from "../../components/EditList";

const List = () => {
  // Set the page title and position using the useEffect hook
  useEffect(() => {
    document.title = `Pacific Life â€¢ News`;
    window.scrollTo(0, 0);
  });

  return <EditList />;
};

export default List;
