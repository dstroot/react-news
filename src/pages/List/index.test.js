import React from "react";
import renderer from "react-test-renderer";
import List from "../List";

describe("List", () => {
  it("it should render", () => {
    const component = renderer.create(<List />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
