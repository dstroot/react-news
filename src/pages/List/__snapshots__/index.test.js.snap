// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List it should render 1`] = `
<div
  className="container"
>
  <div
    className="row justify-content-between mt-5"
  >
    <div
      className="col-md-6"
    >
      <h1
        className="display-4"
      >
        News Story List
      </h1>
    </div>
    <div
      className="col-md-3"
    >
      <a
        className="btn btn-primary btn-block mt-3 mb-3"
        href="/create"
        onClick={[Function]}
      >
        <i
          className="fa"
        />
        Add Story
      </a>
    </div>
  </div>
  <div
    className="row"
  >
    <div
      className="col"
    >
      <table
        className="table table-sm"
      >
        <thead
          className="thead-light"
        >
          <tr>
            <th>
              Date
            </th>
            <th>
              Draft
            </th>
            <th>
              Headline
            </th>
            <th>
              Edit
            </th>
          </tr>
        </thead>
        <tbody>
          <tr>
            <td>
              Loading...
            </td>
          </tr>
        </tbody>
      </table>
    </div>
  </div>
</div>
`;
